### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by icueMike**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,9 +24,18 @@ See [CONTRIBUTING.md](/CONTRIBUTING.md) for ways to get started.
 - [dnd kit](https://dndkit.com/)
 - [react-markdown](https://github.com/remarkjs/react-markdown)
 
+## Installing/Updating NextJS:
```
Comment (icueMike) [inline README.md:27]: I can fine tune this section a bit if need be.

File: README.md [hunk 2]
```diff
@@ -24,9 +24,18 @@ See [CONTRIBUTING.md](/CONTRIBUTING.md) for ways to get started.
...
+First step is being sure your running the latest [NEXTJS](https://github.com/vercel/next.js):
```
Comment (waldyrious) [inline README.md:28]: Just a couple wording/capitalization improvements:
```suggestion
The first step is to make sure you're running the latest [Next.js](https://github.com/vercel/next.js):
```

File: README.md [hunk 3]
```diff
@@ -24,9 +24,18 @@ See [CONTRIBUTING.md](/CONTRIBUTING.md) for ways to get started.
...
+
+`npm install next` or `npm update next`
+
+Most likely you should just be able to update here instead of installing as it should already be installed.
```
Comment (waldyrious) [inline README.md:32]: What does "here" refer to in this sentence? That isn't very clear to me.

Also, I'd suggest adding a comma after "installing", to make the sentence structure more evident.

**Comment by waldyrious** [comment]
Btw, I'd suggest using a more descriptive PR title, and adding content to the PR description, with a reference to the issue it addresses (#101)

**Comment by icueMike** [comment]
Sorry, this is my first contribute Ive ever done on github :/

File: README.md (hunk 2 above)
Comment (icueMike) [reply README.md:28]: Great suggestions, thank you

**Comment by waldyrious** [comment]
Oh, in that case, welcome! And for a first contribution, that's not bad at all! Keep it up. 💪

**Pull request closed without merging**